src/wp-content/themes/twentytwentyone/assets/sass/02-tools/mixins.scss

```
@@ -20,32 +20,61 @@
 // Button style
 // - Applies button styles to blocks and elements that share them.
 @mixin button-style() {
-	line-height: var(--button--line-height);
-	color: var(--button--color-text);
+	border: var(--button--border-width) solid transparent;
+	border-radius: var(--button--border-radius);
 	cursor: pointer;
+	display: block;
 	font-weight: var(--button--font-weight);
 	font-family: var(--button--font-family);
 	font-size: var(--button--font-size);
-	background-color: var(--button--color-background);
-	border-radius: var(--button--border-radius);
-	border: var(--button--border-width) solid var(--button--color-background);
-	text-decoration: none;
+	font-weight: bolder;
+	line-height: var(--button--line-height);
 	padding: var(--button--padding-vertical) var(--button--padding-horizontal);
+	text-decoration: none;
+
+	// Standard Button Color Relationship Logic 
+	&:not(:hover):not(:active)  {
```

```suggestion
	&:not(:hover):not(:active) {
```
Removes the extra space